dist/api_data.py

```diff
@@ -630,6 +630,9 @@ def __ge__(self, other):
             seconds to wait between each checkpoint; setting this value above 0 configures
             periodic checkpoints''',
             min='0', max='100000'),
+        Config('precise_checkpoint', 'false', r'''
```

Comment:
Nitpick: this should probably just say `precise` rather than `precise_checkpoint`. The `checkpoint` part is (IMO) clear from it being a `checkpoint` subconfig.

Reply:
done